libdevcrypto/Common.h

```diff
@@ -77,6 +77,9 @@ using AddressHash = std::unordered_set<h160>;
 /// A vector of secrets.
 using Secrets = std::vector<Secret>;
 
+/// Amount of bytes added when encrypting with encryptECIES.
+static const unsigned c_eciesOverhead = 113;
```

Comment:
This file seems a bit too generic for this protocol detail. Can we move it somewhere else?